internal/yarpctest/outboundtest/outbound_event.go

```diff
@@ -105,6 +111,18 @@ type OutboundEvent struct {
 // Call will validate a single call to the outbound event based on
 // the OutboundEvent's parameters.
 func (e *OutboundEvent) Call(ctx context.Context, t require.TestingT, req *transport.Request) (*transport.Response, error) {
+	if e.WantTimeout != 0 {
+		timeoutBounds := e.WantTimeoutBounds
+		if timeoutBounds == 0 {
+			timeoutBounds = testtime.Millisecond * 20
```

Comment:
possibly, this is just the default threshold, and put in place before we had time dilation...